lib/coach/handler.rb

```diff
@@ -43,7 +45,10 @@ def call(env)
           # This way, the last finish_handler.coach event will have all the details.
           status = response.try(:first) || STATUS_CODE_FOR_EXCEPTIONS
           event.merge!(
-            response: { status: status },
+            response: {
+              status: status,
+              exception: e,
```

Comment:
you can access the variable local to the `rescue` block in the `ensure`? that's so weird!